tests/components/panasonic_viera/test_init.py

```diff
@@ -58,6 +100,37 @@ async def test_setup_entry_encrypted(hass):
         await hass.config_entries.async_setup(mock_entry.entry_id)
         await hass.async_block_till_done()
 
+        assert mock_entry.data[ATTR_DEVICE_INFO] == MOCK_DEVICE_INFO
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)